node/narwhal/src/gateway.rs

```diff
@@ -108,6 +99,34 @@ pub struct Gateway<N: Network> {
     handles: Arc<Mutex<Vec<JoinHandle<()>>>>,
 }
 
+// Dynamic rate limiting.
+impl<N: Network> Gateway<N> {
+    /// The current maxiumum committee size.
+    fn max_committee_size(&self) -> usize {
+        self.storage.current_committee().num_members()
```

Comment:
This will incur a notable perf hit, mainly because it hits the db to fetch the committee

Reply:
Perhaps we just set a hardcoded number for now? Not ideal, but perhaps `200` will be acceptable